apptemplates/test/settings.py

```diff
@@ -22,9 +22,11 @@
             'BACKEND': 'django.template.backends.django.DjangoTemplates',
             'OPTIONS': {
                 'loaders': [
-                    'apptemplates.Loader',
-                    'django.template.loaders.filesystem.Loader',
-                    'django.template.loaders.app_directories.Loader',
+                    ('django.template.loaders.cached.Loader', (
+                        'apptemplates.Loader',
+                        'django.template.loaders.filesystem.Loader',
+                        'django.template.loaders.app_directories.Loader',
+                    )),
```

Comment:
I don't understand why we now have a tuple here, with the original list encapsulated as a nested tuple. Can you explain?

Reply:
It's probably somewhat related to what you explain in [Django PR #9427](https://github.com/django/django/pull/9427/files), but why did it work in the past with a simple list?

Reply:
Okay, I think I get it: It is really just an example that should fail for the old code.

The problem I see with this test app example code is it may only work for the cached loader now, and issues with a regular list of loaders may slip through undetected.

As I see it now, we need an additional test [in the test suite](https://github.com/bittner/django-apptemplates/blob/master/tests.py), e.g. `test_cached`, that overrides the `TEMPLATES[0]['OPTIONS']['loaders']` as you [hard-coded it](https://github.com/dracos/django-apptemplates/blob/522ea9a90429fefc04d039288f46c935b8618ef9/apptemplates/test/settings.py#L24-L30) for running the tests. That test should then clearly be run for Django 1.8+ only.

Reply:
Yes, I should have done a new test, sorry, will do that now.